card/mod/05_standard/set/type/search_type.rb

```diff
@@ -217,7 +216,7 @@ def default_search_params
     total = card.count search_params
     return '' if limit >= total # should only happen if limit exactly equals the total
 
-    @paging_path_args = { :limit => limit, :item=> inclusion_defaults[:view] }
+    @paging_path_args = { :limit => limit, :item=> inclusion_defaults(card)[:view] }
```

Comment:
Right, I was trying to say that the latest code introduces this, and I think it's a bug.  As a rule, regardless of what kind of setting we're talking about, nest should override rule should override code.  But, as written, I think we have a case where the rule overrides the nest.

We had to get this out quickly for Wikirate purposes, so we merged it in before really perfecting it.  I know that's not good form; will try to avoid that in the future.